src/server/js/schema.js

```diff
@@ -1,14 +1,41 @@
 import { merge } from 'lodash';
 import { makeExecutableSchema } from 'graphql-tools';
 
-import {
-    typeDefs as Health,
-    resolvers as healthResolvers,
-} from './resources/health/schema';
+import HouseModel from './resources/houses/model';
+import * as houseSchema from './resources/houses/schema';
+import UserModel from './resources/users/model';
+import * as userSchema from './resources/users/schema';
+
+const queryTypeDefs = `
+    type Query {
+        allHouses: [House],
+        housesByAdministrator(administratorId: ID!): [House],
+        allUsers: [User],
+        userByEmail(email: String): User,
+    }
+`;
+
+const queryResolvers = {
+    Query: {
+        allHouses: (root, args, context, info) =>
+            HouseModel.find().populate('administrators'),
+        housesByAdministrator: (root, args, context, info) =>
+            HouseModel.find({ administrators: args.administratorId }).populate(
+                'administrators'
+            ),
```

Comment:
we might not need to expand the `administrators` field, could possibly just leave it as an array of ids. in that case, remove the populate here, and update the type def for `House.administrators` field to be an array of `ID!` instead of `User`.